bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -578,114 +457,77 @@ All these channels except welcomePersonAtHome are read only.
 ## things/netatmo.things
 
 ```
-// Bridge configuration:
-Bridge netatmo:netatmoapi:home "Netatmo API" [ clientId="*********", clientSecret="**********", username = "mail@example.com", password = "******", readStation=true, readThermostat=false] {
-    // Thing configuration:
-    Thing NAMain inside "Netatmo Inside"       [ id="aa:aa:aa:aa:aa:aa" ]
-    Thing NAModule1 outside "Netatmo Outside"  [ id="bb:bb:bb:bb:bb:bb", parentId="aa:aa:aa:aa:aa:aa" ]
-    Thing NAModule3 rain "Netatmo Rain"        [ id="cc:cc:cc:cc:cc:cc", parentId="aa:aa:aa:aa:aa:aa" ]
+Bridge netatmo:NAAccount:home "Netatmo API" [clientId="", clientSecret="", username="", password=""] {
```

Comment:
Maybe `"Netatmo API"` should be `"Netatmo Account"`?

Reply:
Done